models/netsuite2/src_netsuite2.yml

```
@@ -251,336 +299,293 @@ sources:
           - name: type
             description: The entity type (Contact, CustJob, Job, etc).
           - name: vendor
-            description: The unique identifier of the vendor associated with the entity.
+            description: Th "{{ doc('vendor_id') }}"e unique identifier of the vendor associated with the entity.
```

```suggestion
            description: The "{{ doc('vendor_id') }}" unique identifier of the vendor associated with the entity.
```

should this also just be `"{{ doc('vendor_id') }}" `?

---

Ah this was a find and replace error. It should actually not be here at all. I will remove it.